package.json

```diff
@@ -42,6 +42,7 @@
 		"chart.js": "^2.9.4",
 		"chartjs-node-canvas": "^3.1.0",
 		"chartjs-plugin-datalabels": "^0.7.0",
+		"cheerio": "^1.0.0-rc.12",
```

Comment:
@Prajwal17Tunerlabs where is this package used?

Reply:
this package is used in createSvg api, but currently we are not using this api, when vishnu brought this code to elevate-project there was dependecy on this package, so its only installed as of now